pkg/cmd/task/delete.go

```diff
@@ -60,11 +60,22 @@ tkn t rm foo -n bar
 				Err: cmd.OutOrStderr(),
 			}
 
+			cs, err := p.Clients()
+			if err != nil {
+				return fmt.Errorf("failed to create tekton client")
+			}
+
+			// Check if namespace exists. Return error if namespace specified with -n doesn't exist or if user doesn't have permissions to view.
```

Comment:
it may be better to move these validations to a package `validate`  so that the call appears

```
if err := validate.NamespaceExists(kubeClient, ns); err != nil{
  return err
}
```
WDYT?

Reply:
I have moved the validation code under `pkg/helper/validate`. Let me know what you think.